### PR TITLE
Add method to list media for iOS Simulator

### PR DESCRIPTION
**Pull request by sergey-plevako-badoo**
Added a method to list the files in data/Media/DCIM/100APPLE which shows the number of media added by `xcrun simctl addmedia` command

### CODE REVIEW

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt [hunk 1]
```diff
@@ -28,6 +29,11 @@ class Media(
         restartAssetsd()
     }
 
+    fun list() : String {
+        val listCmd = listOf("ls", "$mediaPath/DCIM/100APPLE")
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/data/Media.kt]: Let's use single column formatted output of 'ls'

please add argument "-1"

File: device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/MediaDto.kt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.badoo.automation.deviceserver.data
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+class MediaDto(
+    @JsonProperty("media")
+    val url: String)
```
Comment (fr0l) [inline device-server/src/main/kotlin/com/badoo/automation/deviceserver/data/MediaDto.kt]: property name is 'url' , in fact it is a string with file names, that are delimited by 'newline' symbol.

1. let's rename the property to something more relevant.
2. let's return a list of file names - i.e. actually a list of strings, so clients do not have to guess anything.

**Review by fr0l — CHANGES_REQUESTED** [review]

**Comment by NickAb** [comment]
You might want to take a look at https://github.com/NickAb/ios-utils.

- It is a CLI tool that runs on a simulator and lets you list and delete media and pasteboards.
- It uses mostly public iOS API, i.e. should be much more stable and less fiddly than working with an underlying simulator file system.
- The only private API used is entitlements keys which are unlikely to change and are the same as ones used by system apps on iOS.

**Comment by sergey-plevako-badoo** [comment]
Updated in aa2762d

**Pull request merged**